### DISCUSSION TITLE
ERROR: Could not resolve SVG file

### DISCUSSION THREAD

**Discussion by mustaqball** [remix-run/remix#5114] [Q&A]
Remix v1.10.1

When building the project it gives an error. Worked fine before.

✘ [ERROR] [plugin css-file] Build failed with 5 errors:
ERROR: Could not resolve "../../images/site/auth/google.svg"

app/routes/signUp/index.jsx:14:25:
import stylesSignup from '../../styles/build/signup/signup.css';
This error came from the "onLoad" callback registered here:
node_modules/@remix-run/dev/dist/compiler/plugins/cssFilePlugin.js:58:12:
build.onLoad({

**Comment by jamesarosen** [comment]
I have occasionally experienced this when changing git branches. If a file exists on one branch, but not another, I sometimes have to restart the Remix dev process to see the change.
